auth0/build.gradle

```diff
@@ -68,21 +71,34 @@ android {
 }
 
 dependencies {
-    implementation 'com.android.support:appcompat-v7:28.0.0'
-    implementation 'com.android.support:customtabs:28.0.0'
+    implementation 'androidx.appcompat:appcompat:1.1.0'
+    implementation 'androidx.browser:browser:1.0.0'
     implementation 'com.squareup.okhttp:okhttp:2.7.5'
     implementation 'com.squareup.okhttp:logging-interceptor:2.7.5'
-    implementation 'com.google.code.gson:gson:2.8.5'
-    implementation 'com.auth0.android:jwtdecode:1.3.0'
+    implementation 'com.google.code.gson:gson:2.8.6'
+    implementation 'com.auth0.android:jwtdecode:2.0.0'
 
-    testImplementation 'junit:junit:4.12'
+    testImplementation 'junit:junit:4.13'
     testImplementation 'org.hamcrest:java-hamcrest:2.0.0.0'
     testImplementation 'org.powermock:powermock-module-junit4:1.6.5'
     testImplementation 'org.powermock:powermock-module-junit4-rule:1.6.5'
     testImplementation 'org.powermock:powermock-api-mockito:1.6.5'
     testImplementation 'org.mockito:mockito-core:1.10.19'
     testImplementation 'com.squareup.okhttp:mockwebserver:2.7.5'
     testImplementation 'com.jayway.awaitility:awaitility:1.7.0'
-    testImplementation 'org.robolectric:robolectric:4.0.2'
-    testImplementation 'com.android.support.test.espresso:espresso-intents:3.0.2'
-}
\ No newline at end of file
+    testImplementation 'org.robolectric:robolectric:4.3.1'
+    testImplementation 'androidx.test.espresso:espresso-intents:3.2.0'
+}
+
+tasks.withType(Test) {
+    jacoco.includeNoLocationClasses = true
```

Comment:
Maybe this is something to add to all libraries through the plugin instead? https://github.com/auth0/oss-library-gradle-plugin/

Reply:
I'd be happy to, do you want me to open a PR on the plugin?

Reply:
Yes, that will do. Note the plugin has 2 plugins, one for Java and for Android.